lib/unifex/code_generator.ex

```diff
@@ -52,7 +52,7 @@ defmodule Unifex.CodeGenerator do
     |> Enum.map(&interface_generator/1)
   end
 
-  @spec bundlex_interface(Bundlex.Native.interface_t()) :: Specs.interface_t()
+  @spec bundlex_interface(:cnode | :nif | :port) :: CNode | NIF | Port
```

Comment:
```suggestion
  @spec bundlex_interface(Bundlex.Native.native_interface()) :: Specs.interface_t()
```

CI was failing because name of the typespec in `Bundlex` has changed, so we should adjust out code to this change, instead of eradicating reference to Bundlex typespec